src/services/GenerateVerifiableCredential.ts

```diff
@@ -167,14 +168,12 @@ export class GenerateVerifiableCredential {
   				case "DOC_NUMBER_INVALID":
   					addToCI("D02");
   					break;
-  				case "TAMPERED":
   				case "DATA_MISMATCH":
```

Comment:
The contra indicator document on confluence has MISSING_HOLOGRAM, NO_HOLOGRAM_MOVEMENT and DATA_MISMATCH as error scenarios.
Should they be handled differently?

Reply:
If the overall suggestion is a REJECT from Yoti for the parent checks then the VC will not be created, if it's an APPROVE but one of these subchecks have failed then my understanding is that we simply don't include a CI but will confirm with BAs